m/maxmin/maxmin_ubuntu_18.04.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Package	: maxmin
+# Version	: 
+# Source repo	: https://github.com/gaearon/react-hot-loader
+# Tested on	: ubuntu_18.04
+# Script License: MIT License
```

Comment:
Script should always be published under the Apache 2.0 license

Reply:
fixed